packages/story-editor/src/components/panels/design/textAccessibility/textAccessibility.js

```diff
@@ -98,11 +84,49 @@ function TextAccessibilityPanel({ selectedElements, pushUpdate }) {
     const newTags = getTextElementTagNames(selectedTextElements, value);
     setCurrentTags(Array.from(newTags.values()));
   };
+
   // Check if tags match for selected item in dropdown
   // If they don't match, we want to show 'Mixed'
   // However, if they match, show the matching value
   const currentValue =
-    new Set(currentTags || []).size === 1 ? currentTags[0] : '((MULTIPLE))';
+    new Set(currentTags || []).size === 1 ? currentTags[0] : MULTIPLE_VALUE;
+
+  const isIndeterminate = MULTIPLE_VALUE === currentValue;
+
+  const tagName = getCommonValue(selectedElements, 'tagName', 'auto');
```

Comment:
Doesn't this do the same thing as the unique set operation above? They seem to overlap a bit? Or is the `currentValue` based on the automatically calculated tags and `tagName` is based on the predefined tag stored on the element?

Reply:
> Or is the `currentValue` based on the automatically calculated tags and `tagName` is based on the predefined tag stored on the element?

yeah